types/mocks.d.ts

```diff
@@ -29,6 +29,14 @@ declare const __static: string;
 
 // Support import for custom module extensions
 // https://www.typescriptlang.org/docs/handbook/modules.html#wildcard-module-declarations
+declare module "*.module.scss" {
+  const classes: { [key: string]: string };
+  export default classes;
+}
+declare module "*.module.css" {
```

Comment:
Normal css-modules should be allowed too, but let's stick to `component.module.scss`